### DISCUSSION TITLE
Manual confirmation of changes.

### DISCUSSION THREAD

**Discussion by timon2013** [serverless/serverless#11407] [Q&A] [answered]
Hi, I'm looking for some good approach to deployment production release on AWS cloud using serverless framework.

In AWS before deploy I want to create "change set", look on this and after manual confirmation deploy this.

It is possible to do this? I'm using Serverless Framework v3.

Something like this:
1. sls deploy -s prod --prepare_change_set
2. I want to confirm this changes manually. 
3. Deploying the changes.

**Comment by pgrzesik** [comment, marked as answer]
Hey @timon2013 - at the moment it's not possible. Framework can use change sets internally, but there's not way to first prepare a change set, inspect it, and deploy it after inspection.

**Reply by timon2013** [reply]
Thanks for your response. It would be a useful feature. This is the only thing I miss. Is there any plan to do this?

**Reply by pgrzesik** [reply]
Hey @timon2013 - I could definitely see it being a useful feature. I believe there are no plans to have it implemented right now, but maybe @medikoo can shed a bit more light

**Reply by medikoo** [reply]
@timon2013 at this point there are no plans to that, but since Framework moved to changesets, it's possible to introduce dry (or pre-confirmed) deployment. If you're interested, please open a normal issue request (as I checked, there seem none opened, which surprised me)

**Reply by timon2013** [reply]
Thanks for help, I have created this issue https://github.com/serverless/serverless/issues/11413
